scss/grid/_classes.scss

```diff
@@ -150,15 +156,29 @@
     }
 
     // Positioning
-    .#{$-zf-size}-#{$center} {
-      @include grid-col-pos(center);
+    @if $center {
+      .#{$-zf-size}-#{$center} {
+        @include grid-col-pos(center);
+      }
     }
 
     // Gutter adjustment
-    .#{$-zf-size}-#{$uncenter},
-    .#{$-zf-size}-#{$push}-0,
-    .#{$-zf-size}-#{$pull}-0 {
-      @include grid-col-unpos;
+    @if $uncenter {
+      .#{$-zf-size}-#{$uncenter} {
+        @include grid-col-unpos;
+      }
+    }
+
+    @if $push {
+      .#{$-zf-size}-#{$push}-0 {
+        @include grid-col-unpos;
+      }
+    }
+
+    @if $pull {
+      .#{$-zf-size}-#{$pull}-0 {
+        @include grid-col-unpos;
```

Comment:
`grid-col-unpos` will generate the same following 4-lines each time it is called:
```css
  position: static;
  float: float;
  margin-right: 0;
  margin-left: 0;
```

So we should rather try to generate a selector with or without uncenter/push/pull classes, instead of duplicating the CSS.

Something like: 
```scss
$selector: null;
$selector: if($uncenter, '#{$selector}.#{$-zf-size}-#{$uncenter} ,', $selector);
$selector: if($push, '#{$selector}.#{$-zf-size}-#{$push} ,', $selector);
$selector: if($pull, '#{$selector}.#{$-zf-size}-#{$pull} ,', $selector);

#{$selector} {
  @include grid-col-unpos;
}
```
But with some utilities to be more readable.
See: https://www.sassmeister.com/gist/dde2d7e79198b0cfab25e9ec54340b95